dasharo-compatibility/network-boot-utilities.robot

```diff
@@ -141,3 +141,16 @@ NBT007.001 Change netboot URL option works correctly
     Enter Submenu From Snapshot    ${ipxe_menu}    Change Netboot iPXE Payload URL
     ${out}=    Read From Terminal Until    Reset to Default
     Should Contain    ${out}    http://boot.dasharo.com/dts/dts.ipxe
+
+NBT008.001 Autoboot disable works correctly
```

Comment:
```suggestion
NBT008.001 Autoboot is disabled
```

`Autoboot disable` suggests that this is setting an option `Autoboot` to `disable`, which it isn't